### PR TITLE
add docplex==2.19.202 and tqdm

### PR DESCRIPTION
**Pull request by toncho11**

### CODE REVIEW

**Comment by toncho11** [comment]
#32

**Comment by gcattan** [comment]
Looks good :)
We can try to test this.
Can you create a file inside the `tests` folder (e.g. `test_docplex.py`), and put this method inside:

```
from docplex.mp.vartype import ContinuousVarType
from docplex.mp.model import Model

def test_get_square_cont_var():
    channels = range(4)
    prob = Model()
    ContinuousVarType.one_letter_symbol = lambda _: 'C'
    return prob.continuous_var_matrix(keys1=channels, keys2=channels,
                                      name='test', lb=-prob.infinity)
```

**Comment by toncho11** [comment]
I added the test,

File: tests/test_docplex.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from docplex.mp.vartype import ContinuousVarType
+from docplex.mp.model import Model
+
+def test_get_square_cont_var():
+    channels = range(4)
+    prob = Model()
+    ContinuousVarType.one_letter_symbol = lambda _: 'C'
+    return prob.continuous_var_matrix(keys1=channels, keys2=channels,
+                                      name='test', lb=-prob.infinity)
```
Comment (gcattan) [inline tests/test_docplex.py:9]:
```suggestion
    mat = prob.continuous_var_matrix(keys1=channels, keys2=channels,
                                      name='test', lb=-prob.infinity)
    assert not mat is None
```

**Comment by toncho11** [comment]
I will try a second PR.

**Pull request closed without merging**